deploy/examples/keycloak/keycloak-with-vault.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keycloak-vault
+type: Opaque
+stringData:
+  test-realm_bindCredential: "test-password"
```

Comment:
Should this be ldap-realm_bindCredential?

Reply:
It should. Thanks for pointing that out. I fixed it.